_posts/2022-12-21-trino-summit-2022-astronomer-recap.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "Using Trino with Apache Airflow for (almost) all your data problems"
+author: "Philippe Gagnon, Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/astronomer.jpg
+---
+
+As we close in on the final talks from [Trino Summit 2022]({% post_url 
+2022-11-21-trino-summit-2022-recap %}), this next talk dives into how to set up
+Trino for batch processing. Trino has historically been well-known for
+facilitating fast adhoc analytics queries as opposed to long-running, resource
+intensive batch/ETL queries. This is due to the fact that Trino kills queries
+that run out of resources in order to prioritize faster query execution. Earlier
+this year, Trino added features to better support batch queries with a new 
+[fault-tolerant execution mode](https://trino.io/blog/2022/05/05/tardigrade-launch.html).
+This mode backs up intermediate data during execution time, allowing Trino to
+restart individual query tasks on failure rather than a query stage or the query
+itself.
+
+Batch queries don't typically involve human intervention and run asynchronously.
+These tasks may depend on each other and have a complex workflow. This talk
+describes how to orchestrate this complexity using Airflow's new Trino
+integration to run Trino batch queries to solve (almost) all your data problems.
+
+<!--more-->
+
+{% youtube xKDN7RUJ5i4 %}
+
+<a class="btn btn-pink btn-md" target="_blank" href="/assets/blog/trino-summit-2022/Trino@Astronomer.pdf">
+  Check out the slides!
+</a>
+
+## Recap
+
+In this talk, Philippe describes a fictional trading scenario that initially
```

Comment:
You need to introduce who Philippe is before referring to him by first name alone.

Reply:
+1, the sentence above starting with "This talk describes" is a good place for that introduction